### ISSUE TITLE
LSTM's gradient seems to diverge

### ISSUE THREAD

**Issue by OhataKenji** [OhataKenji/SketchRNN-Julia#9]
Gradient Clipping may help it.

In the paper, the value of gradient clipping was mentioned.

**Comment by OhataKenji** [comment]
Currently, reducing learning rate is helpful

**Comment by OhataKenji** [comment]
Dropout and Grad Clip not enough

**Comment by OhataKenji** [comment]
Applying Gradient Clipping to LSMT and Dense Layers is working!

**Comment by OhataKenji** [comment]
TODO make the clip value changeable

**Comment by OhataKenji** [comment]
Working fine now
